src/devices/DCMotor/DCMotor.cs

```diff
@@ -0,0 +1,86 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.Pwm;
+using System.Device.Pwm.Drivers;
+
+namespace Iot.Device.DCMotor
+{
+    public abstract class DCMotor : IDisposable
+    {
+        protected GpioController _controller;
+
+        protected DCMotor(GpioController controller)
```

Comment:
NIT: default this to null so people can use a parameter-less controller instead of passing null?

Reply:
Nvm, just realized this won't be public facing.